examples: getting-started: bump Cilium docker image to 1.9

jibi opened the pull request:
Bump cilium docker image used in the getting started [guide](https://docs.cilium.io/en/v1.9.0-rc2/gettingstarted/docker/) from `1.7` to `1.9`.
I tried again the tutorial with the new version and steps stand correct

jibi committed:
examples: getting-started: bump Cilium docker image to 1.9

For #13627

Signed-off-by: Gilberto Bertin <gilberto@isovalent.com>

rolinh commented on examples/getting-started/Vagrantfile:
```diff
@@ -3,9 +3,9 @@
 Vagrant.require_version ">= 2.0.0"
 
-cilium_version = (ENV['CILIUM_VERSION'] || "v1.7.2")
+cilium_version = (ENV['CILIUM_VERSION'] || "v1.9")
```
The environment variables `CILIUM_VERSION` and `CILIUM_TAG` are never defined nor explained in the guide. It probably makes sense to explain the and maybe update `vagrant up` instructions for `CILIUM_VERSION=1.9 vagrant up` or something.

Also, I think that occurrences of `CILIUM_TAG` could actually be replaced by `CILIUM_VERSION` and `CILIUM_TAG` subsequently be removed altogether.

jibi replied:
Updated the `Vagrantfile` and the docs and tested with:
```
$ CILIUM_VERSION=v1.8 vagrant up
..
$ vagrant ssh
..
vagrant@vagrant:~/cilium/examples/getting-started$ cilium version
Client: 1.8.4 c2442da08 2020-10-20T11:13:53+02:00 go version go1.14.10 linux/amd64
Daemon: 1.8.4 c2442da08 2020-10-20T11:13:53+02:00 go version go1.14.10 linux/amd64
```

jibi committed:
examples: getting-started: get rid of cilium_tag in Vagrantfile

This commit unifies the `cilium_version` and `cilium_tag` variables in
the Vagrantfile for the getting started tutorial into a single variable,
`cilium_version`.

Signed-off-by: Gilberto Bertin <gilberto@isovalent.com>
